wavpack/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/local/bin/bash
```

Comment:
Is this an accidental change? Or could you make it smarter?

Reply:
@mherger Not an accidental change, but I realize now it probably breaks non-FreeBSD builds. FreeBSD does not include bash in the basr system. I'll fix it tonight, probably to use env.

Reply:
Corrected with d318170.